packages/core/src/helpers/index.ts

```diff
@@ -2,3 +2,4 @@ export * from './border-radius/border-radius';
 export * from './color/color';
 export * from './font/font';
 export * from './space/space';
+export * from './switch-theme/switchTheme';
```

Comment:
Probably this file should be `switch-theme.js` to align with other helpers?

Reply:
We have `utils/dashCase` and `utils/toCSS` though.
I guess this is as good a time as any to standardise file names.
Do you prefer `dash-case` or `camelCase`?
Should we match file name and its main export?

I'm actually leaning towards just matching folder/file/export name, thus `camelCase` makes more sense for TS.

Reply:
Originally idea was that:

- `helpers` are for mixed Sass/JS, that's why such naming.
- `mixins` for internal use, only Sass
- `utils` for internal use, only JS

Note JS is also TS here.

Now with this PR it's sort of mixed again... If you follow recomendation and split back to `utils` and `mixins`, then `helpers` could stay the same althought `switchTheme` is JS (TS) only.

Reply:
I see. My issue with `mixins` is that we could potentially have `@function`s in there too 🤔 

I'll revert to split `mixins` and `utils` for now, as my main concern is the `core` folder.

Don't understand the bit about `switchTheme`, do you mean it should be in `utils`?
'Cause it's exported 🤔

Reply:
No, I mean `switchTheme` is only a JS helper, there's no Sass one. However, still I think matching naming as `switch-theme.js` is better for it?

Altough, why didn't you keep it under `themes`? I don't see it as a helper for core usage, like building the UI properties. Ahhh, this is why having `core` made sense, as it's a core items of Castor, rest being anything else like theme related.

Maybe simpliest would have been to rename `core` directory to `base` or alternative?

Reply:
I see. If we consider `helpers` as the core selectors, `switchTheme` really doesn't fit there.

Happy to move it under `themes`, let's see how that looks.

Reply:
IMO looks better, however I'd personally name as:

```
/themes/switchTheme/
  - switchTheme.spec.ts
  - switchTheme.ts
```

Reply:
I prefer that too.
Now the only outlier is `border-radius` but I suppose it has a matching mixin in dash-case so we can keep it.